### PR TITLE
Add script to deploy staging applications

### PR DESCRIPTION
**Pull request by beagleknight**
In essence this PR adds the following command: `make deploy_staging`. This command will deploy a new application in Heroku copying the current configuration for the bulletin board in the `staging` environment.

This new application will be added to the pipeline in the `development` stage:
image

This will replace the "Review Apps" since it doesn't work very well with our new docker image.

### CODE REVIEW

**Review by leio10 — APPROVED** [review]
Amazing job!! 💪  🛫

File: Makefile [hunk 1]
```diff
@@ -9,8 +9,10 @@ help:
 	@echo '  test - Run all tests.'
 	@echo 'Releasing packages:'
 	@echo '  release - Bump versions, commit and push changes to the repository and release gems. Requires clean repository and VERSION set.'
+	@echo 'Deploying applications:'
+	@echo '  deploy_staging - Deploy an application to the staging pipeline. Requires heroku login and HEROKU_APP_NAME set.'
```
Comment (orlera) [inline Makefile:13]: `Requires heroku login and HEROKU_APP_NAME set.`
If I am not wrong `HEROKU_APP_NAME` is actually set by `deploy_staging.sh` so setting it before launching the script will have no effect?

**Review by orlera — APPROVED** [review]
🚀

**Review by agustibr — APPROVED** [review]
🚀

**Pull request merged**